File: test/arraysandstrings/PalindromeTest.java
package arraysandstrings;

import org.junit.jupiter.api.Test;

import static arraysandstrings.Palindrome.checkIfPalindrome;
import static org.junit.jupiter.api.Assertions.*;

class PalindromeTest {

    @Test
    void checkIfPalindromePositive() {
        assertTrue(checkIfPalindrome("racecar"));
    }

    @Test
    void checkIfPalindromeNegative() {
        assertFalse(checkIfPalindrome("aceba"));
    }
}

File: src/test/kotlin/arraysandstrings/twopointers/PalindromeTest.kt
package arraysandstrings.twopointers

import kotlin.test.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

class PalindromeTest {
    @Test
    fun checkIfPalindromeTestPositive() {
        assertTrue(Palindrome.checkIfPalindrome("racecar"))
    }

    @Test
    fun checkIfPalindromeTestNegative() {
        assertFalse(Palindrome.checkIfPalindrome("aceba"))
    }
}